app/src/main/java/net/skyscanner/backpack/demo/stories/InteractiveStarRatingStory.kt

```diff
@@ -0,0 +1,27 @@
+package net.skyscanner.backpack.demo.stories
+
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import net.skyscanner.backpack.demo.R
+import net.skyscanner.backpack.starrating.BpkInteractiveStarRating
+
+class InteractiveStarRatingStory : Story() {
+
+  override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+    super.onViewCreated(view, savedInstanceState)
+
+    view.findViewById<BpkInteractiveStarRating>(R.id.star_rating_empty).onRatingChangedListener = { current, max ->
```

Comment:
Can we add it for all views and make it a toast instead of a log? It's much easier to test that way.

```
    view as ViewGroup

    (0..(view.childCount -1)).forEach { idx ->
      val child = view.getChildAt(idx)
      if (child is BpkInteractiveStarRating) {
        child.onRatingChangedListener = { current, max ->
          BpkToast.makeText(context, "$current/$max", BpkToast.LENGTH_SHORT).show()
        }
      }
    }
```

Reply:
Sounds like a good idea!

Reply:
Done